ScnSoft-Education/LeetCodeTraining/DivideIntegers/DivideIntegersSolution.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace LeetCodeTraining.DivideIntegers
+{
+    //https://leetcode.com/problems/divide-two-integers/
+    public static class DivideIntegersSolution
+    {
+        public static int Divide(int dividend, int divisor)
+        {
+            if (dividend == 0)
+            {
+                return 0;
+            }
+
+            if (dividend == int.MinValue && divisor == -1)
+            {
+                return int.MaxValue;
+            }
+
+            int counter = 0;
+            bool negativeResult = (dividend < 0 && divisor > 0) ||
+                                  (dividend > 0 && divisor < 0);
+            dividend = dividend > 0 ? dividend -= dividend + dividend : dividend;
```

Comment:
It seems can be simplified as:
dividend = dividend > 0 ? -dividend : dividend;
Is there any definite reason to calculate this in this way?
`dividend = dividend > 0 ? dividend -= dividend + dividend : dividend;`